tests/foreman/ui/test_settings.py

```diff
@@ -180,12 +180,12 @@ def test_positive_host_dmi_uuid_duplicates(session, setting_update):
 
     :CaseImportance: High
     """
```

Comment:
Remove the Bugzilla check from the test case, otherwise, this test case will not execute.

```
@pytest.mark.skip_if_open('BZ:1975713')
@pytest.mark.tier2
@pytest.mark.parametrize('setting_update', ['host_dmi_uuid_duplicates'], indirect=True)
def test_positive_host_dmi_uuid_duplicates(session, setting_update):

```